### ISSUE TITLE
CheckBoxGroup - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4853]
Add Accessibility information to the CheckBoxGroup page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA (tag these with the 'bug label') or anywhere we observe that we are not meeting best practices.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by CillianHPE** [comment]
@jcfilben 

I have completed a first draft of the Checkbox group a11y audit as added some comments to the excel sheet

https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=BeRXGs

**Comment by jcfilben** [comment]
Thanks @CillianHPE I added some comments

**Comment by jcfilben** [comment]
CheckBoxGroup is passing wcag AA, no follow on issues. Marking this as done
